pkg/dagger.io/dagger/core/fs.cue

```diff
@@ -92,6 +92,29 @@ import "dagger.io/dagger"
 	dest: string
 }
 
+// Remove file or directory from a filesystem tree
+#RmFile: {
```

Comment:
Okay I'll update it to `#Rm`, thank you :rocket:

Reply:
Fixed in https://github.com/dagger/dagger/pull/2422/commits/e39926bccdb5ab6328c32cfc67f34574111c1d50

Reply:
Not sure about the shortened name since we have a guideline for more explicit names.

Reply:
Yes but currently it's pretty explicit that `#Rm` will delete things in the filesystem.

I agreed that `#RmFile` do not cover the possibility to remove directory. At least, if we want to be really explicit, we could name is `#RmFS` ? Since filesystem can be both file or directory? 

I do not know what is the best

Reply:
I was thinking more in line of `#Remove`.

Reply:
Since `#Rm` == `#Remove`, I'm not sure it's accurate
We use `#Mkdir` instead of `#MakeDirectory` because it's `mkdir` is a shell command, as `rm` is